test/NuGet.Core.Tests/NuGet.PackageManagement.Test/BuildIntegratedTests.cs

```diff
@@ -569,81 +568,83 @@ await nuGetPackageManager.InstallPackageAsync(
             messages[0].Message.Should().Contain("Unable to find package nuget.core with version (>= 91.0.0)");
         }
 
-        [Fact(Skip = "https://github.com/NuGet/Home/issues/10212")]
+        [Fact]
         public async Task UpdateAndRollbackPackage()
         {
             // Arrange
+            using var pathContext = new SimpleTestPathContext();
+            using var testSolutionManager = new TestSolutionManager(pathContext);
             var packageIdentity = new PackageIdentity("nuget.core", NuGetVersion.Parse("91.0.0"));
             var packageIdentity2 = new PackageIdentity("nuget.core", NuGetVersion.Parse("2.8.5"));
             var sourceRepositoryProvider = TestSourceRepositoryUtility.CreateV2OnlySourceRepositoryProvider();
```

Comment:
```suggestion
                var sourceRepositoryProvider = TestSourceRepositoryUtility.CreateSourceRepositoryProvider(new PackageSource(pathContext.PackageSource));
```

Can we make the test not use nuget.org?

Similar to some of the earlier ones, just create the packages locally (in this case it's simpler because of no dependencies)